lib/services/role.go

```diff
@@ -1966,7 +1974,8 @@ func (set RoleSet) checkAccess(r AccessCheckable, mfa AccessMFAParams, matchers
 		if matchLabels {
 			debugf("Access to %v %q denied, deny rule in role %q matched; match(namespace=%v, label=%v)",
 				r.GetKind(), r.GetName(), role.GetName(), namespaceMessage, labelsMessage)
-			return trace.AccessDenied("access to %v denied", r.GetKind())
+			return trace.AccessDenied("access to %v denied. User does not have Teleport role permissions for this access.%v",
```

Comment:
```suggestion
			return trace.AccessDenied("access to %v denied. User does not have Teleport role permissions for this access. %v",
```

Let's put the space here and remove the leading space above.